tests/filesTest.js

```diff
@@ -821,10 +924,15 @@ describe('Main: Currently testing files management,', function () {
               'xmlns:oc': 'http://owncloud.org/ns'
             })
             dMultistatus.appendElement('d:response', '', dResponse => {
-              dResponse.appendElement('d:href', '', '/remote.php/dav/meta/123456789/')
+              dResponse.appendElement('d:href', '',
+                MatchersV3.regex(
+                  `.*\\/remote\\.php\\/webdav\\/${file}`,
```

Comment:
```suggestion
                  `\\/remote\\.php\\/webdav\\/${file}`,
```

Reply:
why is that?
if we run server from sub folder then href will be something like `/sub/remote.php/webdav/....`

Reply:
oh yes it does, I didn't considered that. sorry about that.